features/hashchange.yml

```
@@ -0,0 +1,9 @@
+name: HashChangeEvent
```

We have a few other event features (e.g., scrollend) and we've so far used the event name (as in `window.addEventListener("hashchange", () => …`), not the interface. I think this would be a good fit here too:

```suggestion
name: hashchange
```